server/admin/api.go

```diff
@@ -156,3 +158,21 @@ func (s *Server) apiAccounts(w http.ResponseWriter, _ *http.Request) {
 	}
 	writeJSON(w, accts)
 }
+
+// apiAccountInfo is the handler for the '/account/{account id}' API request.
+func (s *Server) apiAccountInfo(w http.ResponseWriter, r *http.Request) {
+	acct := strings.ToLower(chi.URLParam(r, accountNameKey))
+	acctIDSlice, err := hex.DecodeString(acct)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("could not decode accout id: %v", err), http.StatusBadRequest)
+		return
+	}
+	var acctID account.AccountID
+	copy(acctID[:], acctIDSlice)
```

Comment:
Should check the length of (acctIDSlice).  Sorry everyone, I don't know why I'm using comments instead of review.